tests/Fixer/SpecifyArgSeparatorFixerTest.php

```diff
@@ -0,0 +1,35 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\CodingStandard\Fixer;
+
+use PhpCsFixer\Tokenizer\Tokens;
+use PHPUnit\Framework\TestCase;
+
+class SpecifyArgSeparatorFixerTest extends TestCase
+{
+    /**
+     * @test
+     * @dataProvider provideFiles
+     */
+    public function shouldFixCode(string $inputFile, string $expectedOutputFile): void
+    {
+        $fixer = new SpecifyArgSeparatorFixer();
+        $fileInfo = new \SplFileInfo(__DIR__ . '/Fixtures/' . $inputFile);
+        $tokens = Tokens::fromCode(file_get_contents($fileInfo->getRealPath()));
+
+        $fixer->fix($fileInfo, $tokens);
+
+        $this->assertSame(
+            file_get_contents(__DIR__ . '/Fixtures/' . $expectedOutputFile),
+            $tokens->generateCode()
+        );
+    }
+
+    public function provideFiles()
+    {
+        return [
+            'Correct file should not be changed' => ['Correct.txt', 'Correct.txt'],
+            'Wrong file should be fixed' => ['Wrong.txt', 'Fixed.txt'],
+        ];
```

Comment:
While using dataProvider, it makes sense to use one file per case.
Imagine you have bug in Wrong.txt on line 25 - which one of these cases it?

Reply:
@TomasVotruba True, but there is overhead and is also harder to see what is covered and what not. Maybe rather write inline PHP for each case (as some fixer tests do).

You will see which line of code is broken by phpunit's diff, right?

Reply:
Well, how do you run broken code only?

Reply:
You dont (I guess).. but why would you run just part of tests?
If you are developing something new, you can run single test case or I think even one perticular provider value test case in PHPStorm..

Reply:
I do that too, but how to run specific one line in this single file?